### ISSUE TITLE
No matches found when running via yarn, fine via commandline

### ISSUE THREAD

**Issue by fwielstra** [i18next/i18next-parser#343]
## 🐛 Bug Report

I recently updated Yarn to a very recent version (3.0.0-rc.5), and it may have broken something but I can't figure it out on my own.

When I run `yarn i18next 'src/**/*.{ts,tsx}' [-oc]`, everything works as before. However, I also have an "extract" task in my package.json's scripts, defined as follows:

`"extract": "i18next 'src/**/*.{ts,tsx}' [-oc]",`

When I run this, I get the error `No matches found: "[-oc]"` which seems to imply it tries to use the second argument as the pattern to use.
## Your Environment

- *runtime version*: node v15.11.0, yarn 3.0.0-rc.5, npm 7.6.0
- *i18next version*: i.e. 20.3.1
- *os*: Mac

**Comment by karellm** [comment]
My first guess would be to try this: `"extract": "i18next 'src/**/*.{ts,tsx}' -oc",`. Any reason you are putting the arguments between square brackets?

**Comment by fwielstra** [comment]
Sorry for not following up, my notifications and brain are all over the place.

I included the square brackets because they're in the documentation at https://github.com/i18next/i18next-parser#cli, and they seem to work via the CLI so I made Assumptions that they were required even though I probably should have known better; are the square brackets there to indicate they are optional arguments?

Removed the brackets, all is good now. Weird that it used to work with a previous version of either i18next-parser, Yarn or Node/NPM.
